a10_octavia/controller/worker/controller_worker.py

```diff
@@ -1173,12 +1129,37 @@ def failover_amphora(self, vthunder_id):
                 return
 
             LOG.info("Starting Failover process on %s", vthunder.ip_address)
```

Comment:
The situation for vthunder cannot be found in DB should not happen, the vthunder_id is just get from database in A10HealthManager.health_check() and pass to failover_amphora().
And for vthunder is not found, I think there is nothing we can do. Since all information is in vthunder. For only vhtunder_id, we are not able to find amphora and compute.